experimental/dds/tree2/src/feature-libraries/chunked-forest/codec/compressedEncode.ts

```diff
@@ -473,3 +477,32 @@ class LazyFieldEncoder implements FieldEncoder {
 		return this.encoder.shape;
 	}
 }
+
+export function makeCompressedCodec(
```

Comment:
seems like overkill to stamp each encoded chunk with a version. I think one for the overall scheme we're using is good enough.

Reply:
Moved codecs to separate `uncompressedCodecs.ts` and `compressedCodecs.ts` files